### ISSUE TITLE
Tag-/Nachtmodus wechseln

### ISSUE THREAD

**Issue by StephanieFrancke** [SchweizerischeBundesbahnen/DAS#102]
# Ausgangslage
Als Lokpersonal oder Beobachter möchte ich jederzeit zwischen Tag- und Nachtmodus wechseln können, damit ich die Helligkeit bestmöglich an die äusseren Lichtverhältnisse anpassen kann.
# Weiterführende Links
- https://www.figma.com/design/b4EUSXE2AgrPu5Tv3HyEis/11%3A-Stories-Formulierung?node-id=8094-187992&m=dev&t=OjLVlceyuV4tmaHi-1 
# Out of Scope
- Helligkeit einstellen: https://github.com/SchweizerischeBundesbahnen/DAS/issues/101
# Akzeptanzkriterien
- DAS wird im Modus gemäss Gerät geöffnet.
- Es kann jederzeit zwischen Tag- und Nachtmodus gewechselt werden.
- Wenn der Tagmodus aktiv ist, wird der Nachtmodus anwählbar angezeigt.
- Wenn der Nachtmodus aktiv ist, wird der Tagmodus anwählbar angezeigt.
# Testszenarien
## Vorbedingung:
- Die Fahrordnung ist im Tagmodus geöffnet.
## Durchführung:
- Der Nachtmodus steht zur Auswahl und wird angewählt.
  - Die Fahrordnung wechselt in den Nachtmodus.
- Der Tagmodus steht zur Auswahl und wird angewählt.
  - Die Fahrordnung wechselt in den Tagmodus.
- Es wird zwischen verschiedenen Fahrordnungen und den Einstellungen gewechselt.
  - Es kann jederzeit zwischen Tag- und Nachtmodus gewechselt.

# Team Review
- [x] BA Review (@StephanieFrancke)
- [x] DEV Review (@Grodien)
- [x] PO Review (@marcdettwiler)

**Comment by EverheardofUX** [comment]
@StephanieFrancke @Grodien ready für FG heute. Könnt ihr mal eine Review durchführen?

**Comment by marcdettwiler** [comment]
Hallo @StephanieFrancke und @EverheardofUX
Ich habe uns die User Story zum Review zugewiesen. Bitte prüft mit der Version 0.13.0 die Erfüllung der Akzeptanzkriterien von dieser User Story.
Nach Abschluss von Eurem Review könnt ihr Euch wieder unassignen.
Liebe Grüsse
Marc

**Comment by StephanieFrancke** [comment]
BA Testing done, aus fachlicher Sicht in Ordnung.

Diese zwei Punkte überlasse ich deiner Einschätzung, @EverheardofUX:
- Der blaue Balken vom nächsten Halt ist intensiver als im Figma.
- Die Haltepunkte und der Chevron sind blau anstatt weiss.

**Comment by tobe-official** [comment]
> BA Testing done, aus fachlicher Sicht in Ordnung.
> 
> Diese zwei Punkte überlasse ich deiner Einschätzung, [@EverheardofUX](https://github.com/EverheardofUX):
> 
> * Der blaue Balken vom nächsten Halt ist intensiver als im Figma.
> * Die Haltepunkte und der Chevron sind blau anstatt weiss.

Ich habe mal beide Punkte verbessert 👍 Vielen Dank fürs finden und sagen.

**Comment by marcdettwiler** [comment]
Hallo @tobe-official 
Sind Deine Anpassungen in der neuen Version 0.15.0 enthalten?
Gruss Marc

**Comment by tobe-official** [comment]
> Hallo [@tobe-official](https://github.com/tobe-official) Sind Deine Anpassungen in der neuen Version 0.15.0 enthalten? Gruss Marc

Nein, die Anpassungen werden mit anderen Bug-Fixes kommen. Ich habe gerade noch etwas Probleme mit den KM-Tafeln.

**Comment by marcdettwiler** [comment]
Hallo @StephanieFrancke und @EverheardofUX
Ich habe uns die User Story zum Review zugewiesen. Bitte prüft mit der Version 0.16.0 die Erfüllung der Akzeptanzkriterien von dieser User Story.
Nach Abschluss von Eurem Review könnt ihr Euch wieder unassignen.
Liebe Grüsse
Marc

**Comment by marcdettwiler** [comment]
Hallo @tobe-official 
Sind Deine Anpassungen in der neuen Version 0.16.0 enthalten?
Gruss Marc

**Comment by EverheardofUX** [comment]
@tobe-official bitte Punkte für Halteort Weiss und nicht blau umsetzen. Ausserdem alles i.O.

**Comment by tobe-official** [comment]
Hallo @marcdettwiler 
Ja meine Anpassungen sind jetzt vorhanden, aber wie du sehen kannst muss ich sowieso UX-Technisch noch etwas verbessern.
Gruss, Raphael

**Comment by StephanieFrancke** [comment]
BA Testing done, alles i.O.

**Comment by marcdettwiler** [comment]
PO Testing done

- Akzeptanzkriterien erfolgreich mit Fahrordnungen 1513 und T2 getestet

**Comment by EverheardofUX** [comment]
Alles i.O. :) @marcdettwiler @tobe-official

**Comment by stbruni** [comment]
Blaues Icon für Abgestufte Bahnhofgeschwindigkeiten ist im dunklen Modus schlecht sichtbar, dieses wird wahrscheinlich noch nicht angepasst und hat dieselbe Farbe wie im hellen Modus.

![Image](https://github.com/user-attachments/assets/979dd059-2963-48de-a8dc-318985957557) ![Image](https://github.com/user-attachments/assets/64e4eba3-d57b-4079-a4e0-ad983afc27bf)

**Comment by EverheardofUX** [comment]
Hallo @stbruni 
Ja du hast recht. Die werden in "Silver" geändert ASAP. Danke dir. 
LG

**Comment by StephanieFrancke** [comment]
Die Farbe der blauen Punkte wird mit https://github.com/SchweizerischeBundesbahnen/DAS/issues/907 korrigiert. Daher wird diese Story geschlossen.
